harness/determined/common/storage/base.py

```diff
@@ -2,51 +2,10 @@
 import contextlib
 import os
 import shutil
-import uuid
+import storage_id as _uuid
```

Comment:
What

Reply:
lmao, got a little crazy with the find/replace